nixos/tests/installer.nix

```diff
@@ -552,16 +555,26 @@ in {
           + " mkpart primary 2048M -1s"  # PV2
           + " set 2 lvm on",
           "udevadm settle",
+          "sleep 1",
```

Comment:
Maybe add a comment why the sleep is necessary here.

Reply:
It's already in the commit message.

Also, it's not really "needed".

There's some racyness causing some waiting every now and then, and I at least felt like this will make it less likely to trigger. The test still succeeds, it's just less likely to take longer.